### ISSUE TITLE
Cannot focus link text

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode#172360]
related to https://github.com/microsoft/vscode/issues/167470

* open settings UI
* search for `editor spaces @lang:yaml`
* reveal editor.insertSpaces-setting
* (1) click on the title of the setting -> repeatedly press tab -> all elements can focused
* (2) click on the text of the check box (click again) -> repeatedly press tab -> the link of the check box text isn't focused

https://user-images.githubusercontent.com/1794099/214516095-615cacf0-3511-4f01-8171-0fc5114f836c.mov

**Comment by rzhao271** [comment]
I'm unable to reproduce the issue now
